### DISCUSSION TITLE
Can we perform Inner Join query in serverpod project

### DISCUSSION THREAD

**Discussion by Shubhamdsingh** [serverpod/serverpod#479] [Q&A] [answered]
@yahu1031

**Comment by yahu1031** [comment]
You mean custom queries? Yes. You can.

**Comment by vlidholt** [comment, marked as answer]
There is not yet support for joins in Serverpod. However, you can do it with two queries.
There is an example here: https://docs.serverpod.dev/concepts/database-communication#joining-tables-and-nesting-objects

**Reply by iampopal** [reply]
This link is not valid now?

**Comment by iampopal** [comment]
We need inner join. when id exists in both table just in that case we need to show record of first and second table. How can we do this. any example?
